app/furniture/marketplace/cart/delivery_area_component.html.erb

```diff
@@ -4,18 +4,20 @@
     <%- if delivery_area.present? %>
       <div>
         <%= delivery_area.label %>
-
         <div class="text-sm text-gray-500">
           <%= render(Marketplace::Cart::DeliveryExpectationsComponent.new(cart: cart)) %>
         </div>
-
       </div>
-
       <div class="text-right">
         <%= link_to(t('marketplace.cart.delivery_areas.edit.link_to'), cart.location(:edit, child: :delivery_area)) %>
       </div>
     <%- else %>
-      <%= render "marketplace/cart/delivery_areas/form", cart: cart %>
+      <%- if cart.marketplace.has_single_delivery_area? %>
+        <div><%= single_delivery_area_label %></div>
+        <small>If you are ordering from outside this area then 🤷</small>
```

Comment:
Perhaps "Orders with delivery addresses outside of our delivery area will be canceled." or something